backend/main.py

```diff
@@ -1,49 +1,29 @@
 from fastapi import FastAPI
-from typing import List
-from pydantic import BaseModel
-from datetime import datetime
 
-
-class Task(BaseModel):
-    id: int
-    title : str
-    completed : bool
-
-class Item_todo(BaseModel):
-    id : int
-    created_at : datetime
-    title : str
-    color : str
-    task : List[Task]
+from models import Todo
 
 
 app = FastAPI()
 
-#create new todo
-@app.post("/todo", response_model=Item_todo)
-def create_todo(item:  Item_todo):
-    return item
-
 
-#Get list of todo
-@app.get("/todos", response_model=Item_todo)
+@app.get("/todos", response_model=Todo)
 def get_todos():
-    return Item_todo (
-    id = 1,
-    created_at = 2009/7/12,
-    title = "Compras",
-    color = "Red",
-    task = [{
-        "id": 2,
-        "title" : "compras",
-        "completed" : True,
-    },
-    {
-        "id": 3,
-        "title" : "ventas",
-        "completed" : False,
-    }
-    ]
-)
-
-
+    """Get a list of todos."""
+    return Todo(
+        id=1,
+        created_at=12 / 7 / 2007,
```

Comment:
for this date, is not better use datetime.now ? its better than hardcode data

Reply:
date.time sets the real created date?

Reply:
datetime.now sets the current date

Reply:
```python
from datetime import datetime
datetime.now()
# datetime.datetime(2023, 2, 16, 23, 0, 36, 706646)
```